<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Talks | ICH Computational Biology Club </title> <meta name="author" content="ICH Computational Biology Club "> <meta name="description" content=""> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ucl-cbc.github.io/talks/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> ICH Computational Biology Club </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">Home </a> </li> <li class="nav-item active"> <a class="nav-link" href="/talks/">Talks <span class="sr-only">(current)</span> </a> </li> </ul> </div> </div> </nav> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Talks</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="April25" class="col-sm-8"> <div class="title">Detection of Epileptogenic Focal Cortical Dysplasia Using Graph Neural Networks: A MELD Study</div> <div class="author"> Mathilde Ripart (King’s College London) </div> <div class="periodical"> <em></em> Apr 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Question: Can the diagnosis of epilepsy-causing focal cortical dysplasias (FCD) be improved using state-of-the-art artificial intelligence? <br> Findings: In this diagnostic study of 703 patients with epilepsy due to FCD, a context-aware graph neural network (MELD Graph) detected 64% of lesions previously missed by radiologists with a high positive predictive value. Interpretable reports may facilitate clinical integration by characterizing lesion location, size, and morphology alongside the algorithm’s prediction confidence. <br> Meaning: This publicly available interpretable algorithm, validated on a large cohort, can advance the use of AI–based radiological adjuncts for early detection and neurosurgical planning in patients with focal epilepsy due to FCD. </p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="January25" class="col-sm-8"> <div class="title">Deep learning-based predictions of gene perturbation effects do not yet outperform simple linear methods</div> <div class="author"> Constantin Ahlmann-Eltze (UCL Cancer Institute) </div> <div class="periodical"> <em></em> Jan 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Recently, three deep learning methods, partly inspired by the success of ChatGPT, have promised to accurately predict the outcome of unseen perturbation experiments. Such results would drastically reduce the costs of studying gene regulatory networks and could open new avenues for drug research. To see if the proposed models already achieved this goal, we benchmarked them against two deliberately simple baselines. We show that the deep learning methods perform worse than our simple baselines. While the potential for deep neural networks to learn the complex relations in biological systems is plausible, our work highlights the need for critical benchmarking to direct research efforts that aim to bring transfer learning to biology. In this talk, I will present the results of our recent preprint and will give a brief overview on my past and present research.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="November24" class="col-sm-8"> <div class="title">Reading the T cell receptor code</div> <div class="author"> Andreas Tiffeau-Mayer (UCL Division Infection and Immunity) </div> <div class="periodical"> <em></em> Nov 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Antigen-specific T cells play important protective and pathogenic roles in human disease. The hypervariable T cell receptor (TCR) determines specificity, but the computational prediction of specificity from TCR sequence is an unsolved grand challenge in immunology. In my talk, I will discuss some of our recent progress in addressing this challenge: First, I will introduce a population genetics approach to infer correlation functions of specificity on sequence space. Second, I will present our initial attempts at using metric learning to infer which receptors recognize common targets. I will conclude with thoughts on how an ability to read the T cell receptor code will change the way we track immune responses to infection, vaccinations and cancer.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="October24" class="col-sm-8"> <div class="title">AI-based Framework for Predicting KRAS Gene Mutations in Histopathological Images of Colorectal Cancer</div> <div class="author"> Vivek Singh (Queen Mary University London) </div> <div class="periodical"> <em></em> Oct 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Detecting mutations in the KRAS gene is crucial for colorectal cancer patients. The KRAS gene plays a role in the epidermal growth factor receptor signaling pathway, and mutations can impact treatment decisions. Common methods like next-generation sequencing (NGS) are expensive and time-consuming. To address this, we developed KRASFormer, a framework predicting KRAS mutations from widely available H&amp;E-stained images. KRASFormer has two stages: the first filters out non-informative regions, and the second predicts KRAS gene status using a Vision Transformer-based method. The proposed model captures meaningful patterns in tumor tissue, outperforming existing methods on multiple datasets. Our findings highlight the potential of using H&amp;E-stained images for cost-effective and efficient KRAS mutation prediction, the effectiveness of a Transformer-based model, and the importance of collaboration between pathologists and data scientists.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="July24" class="col-sm-8"> <div class="title">Matrix topography induces a cytoskeletal and transcriptional memory supporting metastasis</div> <div class="author"> Oscar Maiques Carlos (Queen Mary University London) </div> <div class="periodical"> <em></em> Jul 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The extracellular matrix (ECM) is a crucial component of solid tumours. Using digital pathology in human and mouse tissues, we define 3 matrix regions: the tumour body (TB), the proximal invasive front (PIF)- and a further distal invasive front (DIF) Increased matrix density coupled to larger fibre thickness, length and alignment are all features of both invasive areas. Importantly, radial fibre orientation is a characteristic of the distal invasive front. Using several in vivo and in vitro 3-dimensional (3D) models, we show that matrix density and alignment induces RhoA/C-ROCK-Myosin activity in cancer cells, while maximal amoeboid directed migration correlates with areas of radial fibre orientation. Importantly, matrix topographical features are enough to prime cancer cells for successful local invasion and metastatic colonisation. Using several spatial OMICs, we describe a mechano-inflammatory transcriptional program linked to matrix topographical features in the invasive fronts of primary tumours that is preserved in metastatic lesions and strongly linked to worse prognosis in patients.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 ICH Computational Biology Club . Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Images from <a href="https://pngtree.com/" target="_blank" rel="external nofollow noopener">pngtree.com</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> </body> </html>